CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 5.0.0
+* **BREAKING**: Old alarms (alarms created pre v5) will be deleted.
```

Comment:
Is there a way to add compatibility with pre v5 ? Like I did in v4 when I added `NotificationSettings` model to `AlarmSettings`. Because otherwise users of all the apps using alarm package will loose their alarms and there's no way to notify them... What do you think ? I can work on that if you want.

Reply:
We might need to do this in multiple languages. Also it's the serialization logic between Kotlin and Dart coupled? It would be great to decouple them: Kotlin saves its own copy of alarms in local storage.

It would be great if you have time to look at this!